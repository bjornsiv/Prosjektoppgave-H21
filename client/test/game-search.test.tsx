import * as React from 'react';
import renderer from 'react-test-renderer';
import GameSearch from '../src/game-search';
import mockGameService from './mock-game-service';

jest.mock('../src/game-service', () => {
  return new mockGameService();
});

describe('Game search component tests', () => {
  test('GameReview search correctly with snapshot', () => {
    const tree = renderer
      .create(<GameSearch match={{ params: { query: "fight" } }} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});