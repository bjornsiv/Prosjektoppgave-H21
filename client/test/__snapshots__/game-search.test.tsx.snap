// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Game search component tests GameReview search correctly with snapshot 1`] = `
<div>
  <div>
    <div
      className="card rounded"
    >
      <div
        className="card-body"
      >
        <p
          className="card-title"
        >
          Search Reaults
        </p>
        <div
          className="card-text"
        >
          <br />
          <div
            className="row"
          >
            <div
              className="col"
            >
              <div
                className="float-start"
              >
                Game title
              </div>
            </div>
            <div
              className="col"
            >
              <div
                className="float-start"
              >
                Description
              </div>
            </div>
            <div
              className="col"
            >
              <div
                className="float-start"
              >
                Genre
              </div>
            </div>
            <div
              className="col"
            >
              <div
                className="float-start"
              >
                Platform
              </div>
            </div>
            <div
              className="col-3"
            >
              <div
                className="float-end"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
